new-lamassu-admin/src/utils/coins/bch.js

```diff
@@ -0,0 +1,15 @@
+const cashaddr = require('cashaddrjs')
```

Comment:
I don't see this added on package.json, this won't really work.

It might be too annoying to fix or move to the server side for all the addresses, remove all validation code.

We can circle back on this if the need arises.

Reply:
all validation code for all coins or just for BCH?